.github/pull_request_template.md

```diff
@@ -1 +1 @@
-<!-- NOTE: All pull requests should have tests (rspec in Ruby, qunit in Javascript). If your code does not include test coverage, please include an explanation of why it was omitted. -->
+<!-- NOTE: All pull requests should have tests (rspec in Ruby, qunit in JavaScript). If your code does not include test coverage, please include an explanation of why it was omitted. -->
```

Comment:
https://github.com/discourse/discourse/pull/12779#discussion_r617129685
This is a brand and this specific instance is user facing (specifically it shows up in the PR I just made).

I skipped the class flavors since I'm not sure if changing them would be problematic.